src/pages/login.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { loginUser } from '../util/loginUser';
+import './loginStyle.css';
+
+const Login = () => {
+  const [isbuttonDisabled, setIsButtonDisabled] = useState(false);
+  const [textUserEmail, setTextUserEmail] = useState('');
+  const [displayErrorMessage, setDisplayErrorMessage] = useState(false);
+  const [isLogin, setIsLogin] = useState(false);
+  useEffect(() => {
+    if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,4})+$/.test(textUserEmail)) {
+      setIsButtonDisabled(false);
+    } else {
+      setIsButtonDisabled(true);
+    }
+  }, [textUserEmail]);
+
+  const handleTextChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTextUserEmail(event.target.value);
+  };
+  const handleClick = async () => {
+    const isExisting = await loginUser(textUserEmail);
+    if (isExisting) {
+      setIsLogin(true);
+      setDisplayErrorMessage(false);
+    } else {
+      setDisplayErrorMessage(true);
+    }
+  };
+  return (
+    <div className='Login'>
+      {isLogin && <Navigate to='/' replace={true} />}
+      <div className='login-form'>
+        <div className='form-content'>
+          <h1>Welcome</h1>
+          <input
+            value={textUserEmail}
+            onChange={(e) => handleTextChange(e)}
+            id='textBoxEmail'
+            type='text'
+            placeholder='user@ravn.co'
+          />
+          <button
+            id='loginButton'

```

Comment:
do you need an id here? or could it just be a class?

Reply:
A class, i wanted to implement ids but i didn't stick to it

Reply:
Makes sense. I think IDs typically are used as a last resort, classes usually are sufficient